tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -188,11 +188,28 @@ class WskWebActionsTests extends TestHelpers with WskTestHelpers with RestUtil w
       action.create(name, file, web = Some("true"))
     }
 
-    val responses = Seq(
-      RestAssured.given().config(sslconfig).options(s"$url.http"),
-      RestAssured.given().config(sslconfig).get(s"$url.json"))
+    Seq(
```

Comment:
Need test that ensures `Access-Control-Request-Headers` isn't echoed back when `web-custom-options` is enabled.